mongodbatlas/data_source_mongodbatlas_third_party_integration_test.go

```diff
@@ -171,7 +150,7 @@ func testAccMongoDBAtlasThirdPartyIntegrationResourceConfig(config *thirdPartyCo
 			config.Name,
 			config.ProjectID,
 			config.Integration.Type,
-			config.Integration.LicenseKey,
+			config.Integration.ServiceKey,
```

Comment:
LicenseKey attribute was for NEW_RELIC integration, PAGER_DUTY integration has [service key](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/1b942e472952fe1a1f9b71e6edf2aa40e04fb56a/mongodbatlas/resource_mongodbatlas_third_party_integration.go#L30)

Reply:
do we need to change any documentation regarding ServiceKey?